charts/osm/values.yaml

```diff
@@ -31,10 +31,10 @@ osm:
 
   # -- `osm-controller` image pull secret
   imagePullSecrets: []
-  # -- Envoy sidecar image for Linux workloads (v1.19.1)
-  sidecarImage: envoyproxy/envoy-alpine@sha256:6502a637c6c5fba4d03d0672d878d12da4bcc7a0d0fb3f1d506982dde0039abd
-  # -- Envoy sidecar image for Windows workloads (v1.19.1)
-  sidecarWindowsImage: envoyproxy/envoy-windows@sha256:c904fda95891ebbccb9b1f24c1a9482c8d01cbca215dd081fc8c8db36db85f85
+  # -- Envoy sidecar image for Linux workloads (v1.21.1)
```

Comment:
We might as well remove the version from these comments if it's included in the value itself.

Reply:
> LGTM, please also update the docs: https://github.com/openservicemesh/osm-docs/blob/97baa0d481928a5607c2dcb3dfb453e93cbf9acd/config.toml#L87

> LGTM, please also update the docs: https://github.com/openservicemesh/osm-docs/blob/97baa0d481928a5607c2dcb3dfb453e93cbf9acd/config.toml#L87

Yes, I have a change that I will push to osm-docs once this merges.